sprokit/src/bindings/python/schedulers/CMakeLists.txt

```diff
@@ -5,4 +5,4 @@ sprokit_create_python_plugin_init(sprokit/schedulers)
 
 sprokit_add_python_module(${CMAKE_CURRENT_SOURCE_DIR}/pythread_per_process_scheduler.py
   sprokit/schedulers
-  pythread_per_process_scheduler)
```

Comment:
You could shorten the name even more since it is largely irrelevant.
"python scheduler", "ppp_sched"

Reply:
removing "_scheduler" was enough to get the file path under the 248 limit  (this name string is used twice in the path).  The "scheduler" part seemed redundant because it is also in a "schedulers" subdirectory, which shows up in the path as well.

Anyway, it works from a compilation point of view, but I was unclear if this name was used elsewhere and if any other code needed to change to make it work at run time.